broker/src/main/java/io/zeebe/broker/clustering/atomix/storage/snapshot/AtomixSnapshotStorage.java

```diff
@@ -146,6 +143,8 @@ public void onNewSnapshot(
     metrics.incrementSnapshotCount();
     observeSnapshotSize(snapshot);
 
+    // todo: simplify now that maxSnapshotCount can only be 1
```

Comment:
Please create a separate issue for this todo and the best would be also to reference it in the todo

Reply:
Will do 👍